### PR TITLE
[fr] Handle negative form and transitive forms for CONFUSION_ER_E_PAR2 ruleset

### PR DESCRIPTION
**Pull request by Sharcoux**
Based on the last diff, handles the following remaining issues:

- sommes can be both the indicative 2 s of "sommer" and the indicative 1 p of "être"
- Handles negative constructions: "tu ne xxx"
- Handles transitive verbs with complement "tu me le xxx"

### CODE REVIEW

**Comment by Sharcoux** [comment]
This is the continuation of the PR: https://github.com/languagetool-org/languagetool/pull/10403

I'll keep tracking down the undesired diffs, but I think we're close to the end.

**Comment by Sharcoux** [comment]
@jaumeortola @LucieSteib Can we merge this? So I can check if every case is 100% handled?

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -54706,15 +54708,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
             <rule>
                 <pattern>
                     <!-- Anything plural, except a verb -->
                     <token postag="[^V].* p$" postag_regexp="yes"/>
                     <marker>
-                        <token regexp="yes">cameras</token>
+                        <token>cameras</token>
                     </marker>
                 </pattern>
                 <message>Cameras est l'indicatif de "se camer". Vouliez-vous dire <suggestion>caméras</suggestion> ?</message>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:54725]: **Formatting:** The target string should also have quotes.I know sometimes it's missing in the existing rules but most of them have this pattern: 
`
            <message>L'usage de la préposition « sur » après le verbe « primer » est déconseillé, vous pouvez aussi choisir la forme « l'emporter sur ».</message>`

**Comment by Sharcoux** [comment]
@LucieSteib I solved your comment and improved the error message. Tell me if anything else is needed. Thanks.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -54722,7 +54722,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                         <token>cameras</token>
                     </marker>
                 </pattern>
-                <message>Cameras est l'indicatif de "se camer". Vouliez-vous dire <suggestion>caméras</suggestion> ?</message>
+                <message>Cameras est l'indicatif du verbe « se camer ». Vouliez-vous dire « <suggestion>caméras</suggestion> » ?</message>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:54725]: **Formatting:** Sorry, I meant on the Cameras in 
`<message>Cameras est l'indicatif du verbe « se camer ».`

**Comment by Sharcoux** [comment]
Should be ok now, right @LucieSteib ?

**Pull request merged**